src/components/newNote/NewNote.jsx

```diff
@@ -0,0 +1,43 @@
+import "./notes.css";
+import { BiPin, BiPalette } from "assets/icons/Icons";
+
+function NewNote() {
+  return (
+    <div className="input-notes-container">
+      <button className="btn-pin" type="button">
+        <BiPin className="fs-m" />
+      </button>
+      <div className="input-notes">
+        <textarea rows="1" placeholder="Title" maxLength="30"></textarea>
+        <textarea rows="1" placeholder="Take notes ...."></textarea>
```

Comment:
Suggestion :- For the take notes text area tag the you can increase the number of rows to 2-3

Reply:
done!